internal/namespaces/instance/v1/custom_marshal.go

```diff
@@ -44,7 +44,6 @@ func init() {
 	human.RegisterMarshalerFunc(instance.ServerLocation{}, serverLocationMarshallerFunc)
 	human.RegisterMarshalerFunc([]*instance.Server{}, serversMarshallerFunc)
 	human.RegisterMarshalerFunc(instance.GetServerResponse{}, getServerResponseMarshallerFunc)
-	human.RegisterMarshalerFunc([]*instance.ServerSummary{}, serverSummariesMarshallerFunc)
```

Comment:
By removing this you probably break list-security group that should display server count in listing

Reply:
Fixed in https://github.com/scaleway/scaleway-cli/pull/600